clang/lib/AST/PrintfFormatString.cpp

```diff
@@ -1044,6 +1117,10 @@ bool PrintfSpecifier::hasValidSpacePrefix() const {
   case ConversionSpecifier::AArg:
   case ConversionSpecifier::FreeBSDrArg:
   case ConversionSpecifier::FreeBSDyArg:
+  case ConversionSpecifier::rArg:
+  case ConversionSpecifier::RArg:
```

Comment:
the space prefix is only valid for signed conversions, so not the capital fixed point conversions.